proto/stride/auction/params.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package stride.auction;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v5/x/auction/types";
+
+// Params defines the defaults for the whole auction module.
+message Params {
+    uint64 defaultSealedAuctionDuration = 1;
```

Comment:
So it would be good for a gov proposal to be able to change this.  An actual auction only lasts a few minutes when kicked off but once started its duration is fixed, this value is a module wide parameter for what the length of the next auction will be.  It would be fine if a gov proposal voted auctions to last shorter or longer going forward and thus it would be fine for this value to get modified while the chain is running.  I'll continue to think about it and research keys.go

Reply:
Gotcha. If these are going to changed by gov proposal, my only suggestion is to remove `default` prefix from these fields.